### ISSUE TITLE
Error thrown when creating second style

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-autoformat#13]
I got the error while typing this text:

![image](https://cloud.githubusercontent.com/assets/156149/20099080/fb9e08d2-a5b6-11e6-9e2d-2b3ce42c35c3.png)

**Comment by szymonkups** [comment]
I took a while to debug this. It looks like it will be a renderer issue. 
When first style is applied, view looks something like this:
```html
<p>You can <strong>bold</strong>[]</p>
```
And inline filler is applied in the second text node (after `</strong>`). When second attribute is applied:

```html
<p>You can <strong>bold</strong> or *italicise*</p>
```
Rendering occurs. First  rendering is checking if selection is in inline filler: https://github.com/ckeditor/ckeditor5-engine/blob/ceb51fa/src/view/renderer.js#L183

`Renderer._isSelectionInInlineFiller` is trying to translate view selection position into DOM position, (by using `DomConverter.viewPositionToDom`),  but this is impossible because:
- in the view we've already changed `*italicize*` to `<em>italicize</em>`,
- there is no DOM corresponding element as it will be created later on during this rendering.
